src/store.js

```diff
@@ -10,30 +10,69 @@ const setProducts = products => ({
   products
 });
 
+const products = (state = [], action) => {
+  switch (action.type) {
+    case SET_PRODUCTS:
+      return action.products;
+    default:
+      return state
+  }
+};
+
+const SET_LINEITEMS = "SET_LINEITEMS";
+
+const setLineItems = items => ({
+  type: SET_LINEITEMS,
+  items
+});
+
+const lineItems = (state = [], action) => {
+  switch (action.type) {
+    case SET_LINEITEMS:
+      return action.items;
+    default:
+      return state;
+  }
+};
+
 export const fetchProducts = () => dispatch => {
   return axios.get('api/products/')
     .then(products => dispatch(setProducts(products.data)))
-}
+};
+
 
 const addProduct = product => dispatch => {
   return axios.post('/api/products/', product)
     .then(() => dispatch(fetchProducts()))
-}
+};
 
 const delProduct = id => dispatch => {
   return axios.delete(`/api/product/${id}`)
     .then(() => dispatch(fetchProducts()))
-}
+};
 
-const reducer = (state = [], action) => {
-  switch (action.type) {
-    case SET_PRODUCTS:
-      return action.products;
-    default:
-      return state;
-  }
+
+const fetchLineItems = () => dispatch => {
```

Comment:
I think you'd need to take the cart Id as an argument to get the lineItems for this particular cart. Otherwise we'd just be getting all lineItems regardless of cart, unless I'm mistaken.

Reply:
I see, there wasn't a route for this on the API server. Ami is adding it.